dashboard/app/controllers/api/v1/pd/applications_controller.rb

```diff
@@ -206,7 +213,10 @@ def search
     private
 
     def get_applications_by_role(role, include_associations: true)
-      applications_of_type = @applications.where(type: TYPES_BY_ROLE[role].try(&:name))
+      # hide incomplete applications for people who are not workshop admins
+      applications_of_type = current_user.workshop_admin? ?
+        @applications.where(type: TYPES_BY_ROLE[role].try(&:name)) :
+        @applications.where.not(status: 'incomplete').where(type: TYPES_BY_ROLE[role].try(&:name))
```

Comment:
i would maybe switch this to 

```suggestion
      applications_of_type = @applications.where(type: TYPES_BY_ROLE[role].try(&:name))
      applications_of_type = applications_of_type.where.not(status: 'incomplete') if current_user.workshop_admin?
```

just to remove that code duplication

(same for above if/else in search)

Reply:
Ah, that's nicer––thanks!